semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -66,7 +68,7 @@ class MarkdownRenderer:
     )
     p_end = re.compile("<!--\\s*endsemconv\\s*-->")
     default_break_conditional_labels = 50
-    valid_parameters = ["tag", "full", "remove_constraints"]
+    valid_parameters = ["tag", "full", "remove_constraints", "metric_table"]
```

Comment:
Is this needed as a parameter? Shouldn't the markdown generator be able to determine this automatically based on the semantic convention group kind?